### PR TITLE
Feat/1073 Restore in progress banner

### PR DESCRIPTION
**Pull request by mattrussell36**
closes: #1073
- add new app banner component
- add logic to fetch network limits and stats endpoints to check for current block height against bootstrap block height

### CODE REVIEW

File: src/app-loader.tsx [hunk 1]
```diff
@@ -99,6 +99,62 @@ export const AppLoader = ({ children }: { children: React.ReactElement }) => {
     }
   }, [appDispatch, ethAddress, vegaKeysLoaded, setAssociatedBalances]);
 
+  React.useEffect(() => {
+    let interval: any = null;
+
+    const getNetworkLimits = async () => {
+      const [networkLimits, stats] = await Promise.all([
+        fetch("https://n04.d.vega.xyz/network/limits").then((res) =>
```
Comment (dexturr) [inline src/app-loader.tsx:107]: Hard coded for dev, intentional?

Same on line 110

File: src/app-loader.tsx [hunk 2]
```diff
@@ -99,6 +99,62 @@ export const AppLoader = ({ children }: { children: React.ReactElement }) => {
...
+      const restoreBlock = Number(
+        networkLimits.networkLimits.bootstrapBlockCount
+      );
+      const currentBlock = Number(stats.statistics.blockHeight);
+
+      if (currentBlock <= restoreBlock) {
+        appDispatch({
+          type: AppStateActionType.SET_BANNER_MESSAGE,
+          message:
+            "The network is less than 900 blocks old, it could be in the process of restoring from a checkpoint",
```
Comment (dexturr) [inline src/app-loader.tsx:122]: minor (non-blocking): translate

**Review by dexturr — COMMENTED** [review]
Hard coded to dev, otherwise LGTM

File: src/app-loader.tsx (hunk 1 above)
Comment (mattrussell36) [reply src/app-loader.tsx:107]: hardcoded just for now. I think Phil is adding it to GQL so waiting on that 😄

Comment (dexturr) [reply src/app-loader.tsx:107]: Ah i didn't realize this was a new API :+1:

Comment (mattrussell36) [reply src/app-loader.tsx]: Updated to use REACT_APP_VEGA_URL with new helper function `getDataNodeUrl`

**Comment by dexturr** [comment]
@mattrussell36 I am going to repoint to develop as branch spaghetti is over

**Pull request merged**